blocks/extras/extensions/plot.py

```diff
@@ -157,9 +166,39 @@ def preexec_fn():
     def __getstate__(self):
         state = self.__dict__.copy()
         state['sub'] = None
+        state['_push_thread'] = None
```

Comment:
Actually, that won't work either, because it may result in a KeyError... `state.pop('_push_thread', None)` will do the job though.